src/library_syscall.js

```diff
@@ -227,9 +227,15 @@ var SyscallsLibrary = {
   _emscripten_syscall_mmap2__deps: ['memalign', 'memset', '$SYSCALLS',
 #if FILESYSTEM && SYSCALLS_REQUIRE_FILESYSTEM
     '$FS',
+#endif
+#if MINIMAL_RUNTIME // MINIMAL_RUNTIME does not have a global PAGE_SIZE runtime variable.
+    'getpagesize'
 #endif
   ],
   _emscripten_syscall_mmap2: function(addr, len, prot, flags, fd, off) {
+#if MINIMAL_RUNTIME // MINIMAL_RUNTIME does not have a global PAGE_SIZE runtime variable.
+    var PAGE_SIZE = _getpagesize();
+#endif
```

Comment:
how about removing the ifdefs for minimal runtime, and always using `_getpagesize()` in both modes? that would keep things simpler overall, and I don't see much downside for the normal mode.

Reply:
(again, my goal is to have as few code differences between the minimal and normal runtime, to reduce complexity and make it easier to migrate to it)

Reply:
Sure, that works - I just did not want to regress code size for the classic runtime with my changes.